corehq/apps/ota/views.py

```diff
@@ -245,3 +247,14 @@ def heartbeat(request, domain, hq_app_id):
         info.update(LatestAppInfo(brief_app_id, domain).get_info())
 
     return JsonResponse(info)
+
+
+@location_safe
+@login_or_digest_or_basic_or_apikey()
+@require_GET
+def get_next_id(request, domain):
+    bucket_id = request.GET.get('pool_id')
+    session_id = request.GET.get('request_id')
```

Comment:
@esoergel We're going with "Session_id" here, right? I'll update my spec.

Reply:
Yup, will update.